catalog/core/catalog-core-commons/src/main/java/ddf/util/XPathHelper.java

```diff
@@ -156,13 +157,16 @@ public static String print(Node n, String encoding) {
       StringWriter stringOut = new StringWriter();
 
       DOMImplementationLS domImpl = (DOMImplementationLS) document.getImplementation();
-      LSSerializer serializer = domImpl.createLSSerializer();
-      LSOutput lsOut = domImpl.createLSOutput();
-      lsOut.setEncoding(encoding);
-      lsOut.setCharacterStream(stringOut);
-
-      serializer.write(n, lsOut);
-
+      LSSerializer lsSerializer = domImpl.createLSSerializer();
+      lsSerializer.getDomConfig().setParameter("xml-declaration", false);
+      LSOutput lsout = domImpl.createLSOutput();
+      lsout.setEncoding(encoding);
+      lsout.setCharacterStream(stringOut);
+
+      NodeList childNodes = n.getChildNodes();
+      for (int i = 0; i < childNodes.getLength(); i++) {
+        lsSerializer.write(childNodes.item(i), lsout);
```

Comment:
This `for loop` is probably worth an inline comment and a reference to the DDF JIRA issue #

Reply:
fixed